app/models/post.rb

```diff
@@ -0,0 +1,7 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments
+
+  validates :title, presence: true, uniqueness: { case_sensitive: false }
+  validates :body, presence: true
```

Comment:
- [ ] Kindly check if the body and title are not blank. Currently, I can have a title and body like `tittle: '', body: ''`.